EuroPythonBot/config.toml

```diff
@@ -29,83 +29,95 @@ LOG_LEVEL = "INFO"
-# 
-# [programme_notifications.rooms.2191.livestreams]
-# # South Hall 2B Livestream URLs
-# "2023-07-19" = "https://europython.eu/live/south-hall-2b"
-# "2023-07-20" = "https://europython.eu/live/south-hall-2b"
-# "2023-07-21" = "https://europython.eu/live/south-hall-2b"
-# 
-# [programme_notifications.rooms.2194]
-# # North Hall
-# webhook_id = "ROOM_2194"
-# discord_channel_id = "1120780401791750315"
-# 
-# [programme_notifications.rooms.2194.livestreams]
-# # North Hall Livestream URLs
-# "2023-07-19" = "https://europython.eu/live/north-hall"
-# "2023-07-20" = "https://europython.eu/live/north-hall"
-# "2023-07-21" = "https://europython.eu/live/north-hall"
-# 
-# [programme_notifications.rooms.2192]
-# # Terrace 2A
-# webhook_id = "ROOM_2192"
-# discord_channel_id = "1120780461195657387"
-# 
-# [programme_notifications.rooms.2192.livestreams]
-# # Terrace 2A Livestream URLs
-# "2023-07-19" = "https://europython.eu/live/terrace-2a"
-# "2023-07-20" = "https://europython.eu/live/terrace-2a"
-# "2023-07-21" = "https://europython.eu/live/terrace-2a"
-# 
-# [programme_notifications.rooms.2193]
-# # Terrace 2B
-# webhook_id = "ROOM_2193"
-# discord_channel_id = "1120780490576777287"
-# 
-# [programme_notifications.rooms.2193.livestreams]
-# # Terrace 2B Livestream URLs
-# "2023-07-19" = "https://europython.eu/live/terrace-2b"
-# "2023-07-20" = "https://europython.eu/live/terrace-2b"
-# "2023-07-21" = "https://europython.eu/live/terrace-2b"
+pretalx_schedule_url = "https://pretalx.com/api/events/pyconde-pydata-2024/schedules/latest/"
+conference_website_session_base_url = "https://2024.pycon.de/program/{slug}"
+conference_website_api_session_url = "https://2024.pycon.de/program/{code}"
+
+[[programme_notifications.notification_channels]]
+webhook_id = "PYTHON_DISCORD"
+include_channel_in_embeds = false
+
+[[programme_notifications.notification_channels]]
+webhook_id = "PROGRAMME_NOTIFICATIONS"
+include_channel_in_embeds = true
+
+# Rooms from pretalx
+# KUPPELSAAL: 2578 in pretalx
+[programme_notifications.rooms.2578]
+webhook_id = "ROOM_2578"
+discord_channel_id = "1225453671152877680"
+
+[programme_notifications.rooms.2578.livestreams]
+# TODO: update URLs
+"2024-04-22" = "https://2024.pycon.de/live"
+"2024-04-23" = "https://2024.pycon.de/live"
+"2024-04-24" = "https://2024.pycon.de/live"
+
+# B09: 2579
+[programme_notifications.rooms.2579]
+webhook_id = "ROOM_2579"
+discord_channel_id = "1225453705910816868"
+
+[programme_notifications.rooms.2579.livestreams]
+# TODO: update URLs
+"2024-04-22" = "https://2024.pycon.de/live"
+"2024-04-23" = "https://2024.pycon.de/live"
+"2024-04-24" = "https://2024.pycon.de/live"
+
+# B07-B08: 2580
+[programme_notifications.rooms.2580]
+webhook_id = "ROOM_2580"
+discord_channel_id = "1225454000800006144"
+
+[programme_notifications.rooms.2580.livestreams]
+# TODO: update URLs
+"2024-04-22" = "https://2024.pycon.de/live"
+"2024-04-23" = "https://2024.pycon.de/live"
+"2024-04-24" = "https://2024.pycon.de/live"
```

Comment:
btw the repo is open so anyone with a github account could get access to the live stream if they want to.

Reply:
Good point. We should think about how to deal with the live stream links when we want to include them